packages/appeals-service-api/src/controllers/local-planning-authorities.js

```diff
@@ -6,27 +6,25 @@
  *
  * Trialist LPAs is our own data
  */
-
-const LPASchema = require('../schemas/lpa');
-const { getLpaList, createLpaList } = require('../services/lpa.service');
+const { getLpaList, createLpaList, getLpaById } = require('../services/lpa.service');
 
 module.exports = {
 	async get(req, res, next) {
 		const { id } = req.params;
 
 		req.log.info({ id }, 'Retrieving LPA');
 
-		const data = await LPASchema.findOne({
+		const lpa = await getLpaById({
 			id: new RegExp(id, 'i')
 		});
 
-		if (!data) {
+		if (!lpa) {
```

Comment:
I'd always encourage positive conditional checks, rather than negations https://www.obsidiansecurity.com/blog/why-to-stop-writing-negative-code/